integration/tests/tracer/test_tracer_debug_methods.py

```diff
@@ -20,12 +22,42 @@ def get_schema(self, file_name):
             d = json.load(f)
             return d
 
-    @pytest.mark.skip(reason="bug NDEV-2196")
-    def test_debug_trace_call(self):
+    def validate_response_result(self, response):
+        schema = self.get_schema("debug_traceCall.json")
+        validator = Draft4Validator(schema)
+        assert validator.is_valid(response["result"])
+
+    def test_debug_trace_call_invalid_params(self):
+        response = self.tracer_api.send_rpc(
+            method="debug_traceCall", params=[{}, "0x0"])
+        assert "error" in response, "No errors in response"
+        assert response["error"]["code"] == -32603, "Invalid error code"
```

Comment:
Did you check code number with go-eth?

Reply:
yes I did, it is jsonrpc standard errors